src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -817,14 +821,27 @@ private JButton getLotButton() {
 			Lot lotToUpdate = selectedInventoryRow.getLot() != null ? selectedInventoryRow.getLot() : null;
```

Comment:
Shouldn't be equal to this?
```suggestion
			Lot lotToUpdate = selectedInventoryRow.getLot();
```